### PR TITLE
feat(print-nuxt): picasso landscape mode + picasso dayResponsibles

### PR DESCRIPTION
**Pull request by usu**
- [x] landscape mode for picasso
- [x] print day responsibles on picasso

This PR is not pagedjs compatible, so this is under the assumption of the comments in #3024

[Picasso-Landscape.pdf](https://github.com/ecamp/ecamp3/files/9731038/Picasso-Landscape.pdf)
[Picasso-Portrait.pdf](https://github.com/ecamp/ecamp3/files/9731039/Picasso-Portrait.pdf)

### CODE REVIEW

File: print/components/PicassoPeriod.vue [hunk 1]
```diff
@@ -65,8 +88,7 @@ export default {
   },
   computed: {
     periodChunks() {
-      // TODO: choose chunkSize based on orientation
-      const chunkSize = 3
+      const chunkSize = this.landscape ? 5 : 3
```
Comment (carlobeltrame) [inline print/components/PicassoPeriod.vue]: In ecamp v2 it was 8 / 4: https://github.com/ecamp/ecamp/blob/devel/src/application/print/class/build_picasso.php#L73..L76
Is that too crowded? I always felt those were quite sensible defaults.

File: print/components/config/Picasso.vue [hunk 2]
```diff
@@ -35,6 +38,8 @@ export default {
     this.periods = this.options.periods.map((periodUri) => {
       return this.$api.get(periodUri) // TODO prevent specifying arbitrary absolute URLs that the print container should fetch...
     })
+
+    this.landscape = this.options.orientation === 'L'
```
Comment (carlobeltrame) [inline print/components/config/Picasso.vue]: Why is this a data property populated in `fetch`? Why not simply a computed property?

File: print/components/PicassoChunk.vue [hunk 3]
```diff
@@ -85,27 +101,33 @@ export default {
 
       return `[${responsibles}]`
     },
+
+    dayResponsiblesCommaSeparated(date) {
+      const day = this.getDayByDate(date)
+      if (!day) return null
+      return dayResponsiblesCommaSeparated(day, this.$tc.bind(this))
+    },
```
Comment (carlobeltrame) [inline print/components/PicassoChunk.vue:106]: You have added this function once in this component and once in print/helpers, but really it should probably be in common and have unit tests, don't you think?

File: print/components/PicassoChunk.vue [hunk 4]
```diff
@@ -21,14 +21,28 @@
...
+          <span class="tw-block">
+            {{ $date.utc(date).format($tc('global.datetime.dateLong')) }}
+          </span>
+
+          <span v-if="hasDayResponsibles(date)" class="tw-text-sm tw-italic">
+            {{ $tc('entity.day.fields.dayResponsibles') }}:
+            {{ dayResponsiblesCommaSeparated(date) }}
+          </span>
+
+          <!-- <day-responsibles :date="date" :period="period" :readonly="!editable" /> -->
```
Comment (carlobeltrame) [inline print/components/PicassoChunk.vue]: Remove?

**Review by carlobeltrame — CHANGES_REQUESTED** [review]

File: print/components/PicassoPeriod.vue (hunk 1 above)
Comment (usu) [reply print/components/PicassoPeriod.vue]: Didn't really give it a lot of thought, to be honest. Can try with 8/4 and see how it looks. Look and feel depends probably a lot on how many parallel events are scheduled.

File: print/components/config/Picasso.vue (hunk 2 above)
Comment (usu) [reply print/components/config/Picasso.vue]: Depends probably on the perspective of "simply": Didn't really need a computed as not reactive anyway. My brain is more wired to fetch and set data, but can do it as computed if this makes it more understandable.

File: print/components/PicassoChunk.vue (hunk 3 above)
Comment (usu) [reply print/components/PicassoChunk.vue:106]: Yeah, probably makes sense to put this in commons and we could then potentially use if here as well:
https://github.com/ecamp/ecamp3/pull/3022/files#diff-fcb428166931633c8e0d0a8df3f6398f533d8406a91d437df0de24a83b0fe8b6R12

File: print/components/PicassoPeriod.vue (hunk 1 above)
Comment (usu) [reply print/components/PicassoPeriod.vue]: Increased to 7 in landscape and 4 in portrait. 8 in landscape seems to me rather dense.

https://github.com/ecamp/ecamp3/pull/3026/commits/83826a275107797778cf59a84d90fd8cfe19a0f0

File: print/components/PicassoChunk.vue (hunk 3 above)
Comment (usu) [reply print/components/PicassoChunk.vue:106]: Move in commit https://github.com/ecamp/ecamp3/pull/3026/commits/50484442f0499ead958511a32abb2a1e962c55c4

File: print/components/PicassoChunk.vue (hunk 4 above)
Comment (usu) [reply print/components/PicassoChunk.vue]: Fixed in https://github.com/ecamp/ecamp3/pull/3026/commits/50484442f0499ead958511a32abb2a1e962c55c4

**Review by carlobeltrame — APPROVED** [review]

**Review by BacLuc — APPROVED** [review]

**Pull request merged**